src/gurt/misc.c

```diff
@@ -1,5 +1,6 @@
 /*
  * (C) Copyright 2016-2024 Intel Corporation.
```

Comment:
Copyright check is complaining because commit is signed with intel
> Signed-off-by: Cedric Koch-Hofer <cedric.koch-hofer@intel.com>

Reply:
Arghhh, I was not understanding why it was complaining as the new HPe copyright was added.
I am going to fix this asap.
Thanks for the info.